core/src/env/srml/types.rs

```diff
@@ -83,3 +89,55 @@ pub type Moment = u64;
 
 /// The default SRML blocknumber type.
 pub type BlockNumber = u64;
+
+/// The default SRML call type.
+#[derive(Encode)]
+#[cfg_attr(feature = "test-env", derive(Decode, Debug, Clone, PartialEq, Eq))]
+pub enum Call {
+    #[codec(index = "5")]
+    Balances(super::calls::Balances<DefaultSrmlTypes>),
```

Comment:
It only features `Balances` atm? Is everything else hard coded later? Are custom runtimes supposed to also configure ink! in a way to extend the set of supported dispatch calls here?